pattern_import_export_custom_header/tests/test_custom_header.py

```diff
@@ -138,3 +141,45 @@ def test_5_add_new_initial_headers(self):
         self.assertEqual(
             self.pattern_config.custom_header_field_name_ids[1].custom_name, "custom_1"
         )
+
+    def test_6_get_data_to_export_with_custom_header(self):
+        new_custom_names = [
```

Comment:
Can you create a more complex test ?
- with a different order (the id of the partner is the first in the partner but is not the first in the custom header)
- with aditionnal header (without data in the line)
I thinks it will not work :)